tasks.py

```diff
@@ -491,9 +495,10 @@ def pre_push(c: Any) -> None:
     if not ret:
         LOG.warning("Run `terraform fmt -recursive` locally to fix formatting")
         sys.exit(1)
-    cmd = "nix fmt"
+    cmd = "alejandra --check ."
```

Comment:
Change this to :

`nix fmt -- --fail-on-change`

Reply:
oh that's better yeah